schema/ingestion_config/v1.0.0/ingestion_config_models_extended.py

```diff
@@ -496,39 +501,46 @@ def validate_publications(publications: Optional[str]) -> Optional[str]:
 # ==============================================================================
 # Sources Validation
 # ==============================================================================
-def validate_sources(source_list: List[DefaultSource] | List[VoxelSpacingSource]) -> None:
+def validate_sources(source_list: List[StandardSource] | List[VoxelSpacingSource]) -> None:
     total_errors = []
 
     # For verifying that all source entries each only have one finder type
     multiple_finders_in_each_source_entries_errors = []
-    standalone_finders_in_each_source_entries_errors = []
+    only_parent_filter_in_source_entry_errors = []
+    only_exclude_in_source_entry_errors = []
 
     for i, source_element in enumerate(source_list):
         finders_in_source_entry = []
         has_parent_filters = False
+        has_exclude = False
         for finder in source_element.model_fields:
             # If the finder is not None, add it to the list of finders in the source entry
             if getattr(source_element, finder) is None:
                 continue
             if finder == "exclude":
-                continue
-            if finder == "parent_filters":
+                has_exclude = True
+            elif finder == "parent_filters":
                 has_parent_filters = True
             else:
                 finders_in_source_entry.append(finder)
         # If there are more than one finder in the source entry, add the source entry index and the finders to the error set
         if len(finders_in_source_entry) > 1:
             multiple_finders_in_each_source_entries_errors.append((i, finders_in_source_entry))
-        elif len(finders_in_source_entry) == 0 and has_parent_filters:
+        if len(finders_in_source_entry) == 0 and has_parent_filters:
```

Comment:
Not necessarily, because parent_filters and exclude are considered entries in the default pydantic model. finders_in_source_entry tracks only actual "finders", e.g. source_glob, destinaton_glob, literal, etc (so when parent_filters / exclude exists, but actual finders don't, the length will still be 0). So in case where there's only parent_filters and/or exclude, it will pass default pydantic validation.